dist/keysim.js

```diff
@@ -1,123 +1,110 @@
-(function (global, factory) {
```

Comment:
If we stop checking `dist` into git, does that also mean we do not publish with bower?

I mean, bower is very simple, in that it just downloads packages.

I've noticed that bower-packages always have a checked in build, which is what is intended to be used by bower package consumers.